labs/devnet-alexa-lab/devnet-alexa-lab.json

```diff
@@ -1,5 +1,5 @@
 {
-  "labId": "devnet-alexa-lab",
+  "redirectTo": "ucsm-powertool-101",
```

Comment:
Keep the labId line, and add the redirectTo as you did.

Reply:
Thanks Anne, merging the changes and closing this pull request.